storage/innobase/buf/buf0dblwr.cc

```diff
@@ -346,7 +346,7 @@ buf_parallel_dblwr_make_path(void)
 	if (parallel_dblwr_buf.path)
 		return(DB_SUCCESS);
 
-	char path[FN_REFLEN];
+	char path[FN_REFLEN + 1 /* OS_PATH_SEPARATOR */];
```

Comment:
Why?

Reply:
The reason is here:
```
			ut_snprintf(path, sizeof(path), "%s%c%s",
				    dir,
				    OS_PATH_SEPARATOR,
				    srv_parallel_doublewrite_path);
```

Reply:
Linux has maximum path of 4096 characters.
`FN_REFLEN` seems to be MySQL limitation.
In this case `FN_REFLEN+1` should work well because after `ut_snprintf(path,...` we have:
```
parallel_dblwr_buf.path = mem_strdup(path);
```